reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-72.md

```diff
@@ -0,0 +1,166 @@
+---
+title: What's New in PowerShell 7.2
+description: New features and changes released in PowerShell 7.2
+ms.date: 09/23/2021
+---
+
+# What's New in PowerShell 7.2
+
+PowerShell 7.2 is the next Long Term Servicing (LTS) release is built on .NET 6.0.
+
+PowerShell 7.2 includes the following features, updates, and breaking changes.
+
+- New universal installer packages for most supported Linux distributions
+- Microsoft Update support
+- 3 new experimental features
```

Comment:
This says 3 but lists two?  The ANSI color support is non-experimental, but the ANSI FileInfo color support is experimental.  Line 16 should probably be "Improve native command argument passing support"

Reply:
Fixed